src/search.ts

```diff
@@ -314,7 +314,11 @@ export default class Search extends SearchUtils implements SearchInterface {
                 return;
             }
 
-            const q = this.constructQuery(query, senderIds, threadIds, fileType);
+            if (!SORT_ORDER || typeof SORT_ORDER !== 'number' || Math.round(SORT_ORDER) !== SORT_ORDER) {
+                SORT_ORDER = searchConfig.SORT_BY_SCORE;
+            }
+
+            const q = this.constructQuery(query, senderIds, threadIds, fileType, SORT_ORDER === 1);
```

Comment:
I see that you are using `SORT_ORDER` as a boolean. Does this identify whether the sort option is `recency` or 'relevancy`? If yes, what if a new sorting option is available? Will this be changed?

Reply:
Currently, we only have two filters `recency` and `relevancy` with option `0` and `1`
and yes we need to change this if there is any new sorting option in the future.

Reply:
I understand that this approach would be slightly more performant. But, I'd suggest to keep it future-proof.

Reply:
@VikasShashidhar I have moved it to const